docs/feb2020-workshop-report.md

```diff
@@ -0,0 +1,226 @@
+  - [Essential for Cylc 8.0](#essential-for-cylc-8)
+  - [Nice to have for Cylc 8.0](#nice-to-have-for-cylc-8)
+  - [For Cylc-8.1+](#for-cylc-8.1+)
+- [Tentative development timeline](#tentative-development-timeline)
+
+## Goal and topics
+
+The primary goal of the workshop was to review Cylc 8 progress and chart the
+roadmap to completion by early 2021.
+
+For specific discussion topics covered during the workshop and links to
+relevant background material see the [Workshop
+Agenda](feb2020-workshop-agenda.md). For additional detail
+see [workshop notes by HO](feb2020-workshop-notes.md) and
+[JR](feb2020-workshop-notes-jr.md).
+
+## Executive summary
+
+Python 2 end of life, obsoletion of the PyGTK GUI toolkit, and a near-universal
+modern preference for web UIs over traditional desktop GUIs, has led to an
+urgent need to re-engineer Cylc to future proof our investment in workflow
+orchestration.
+
+The new [Cylc 8 architecture](cylc-8-architecture.md) was agreed during the
+[December 2018 Cylc workshop](dec-workshop-repord.md) at BOM. Much has been
+achieved since then despite some unexpected resourcing issues (staff turnover
+etc.). At this point the back-end workflow engine and CLI have been migrated to
+Python 3, and the major components Cylc 8 and the connections between them
+are all in place and working. It is already possible to run workflows
+of moderate size with Cylc 8 and view and control them in the new web UI. For
+details, see [Progress to date](#progress-to-date) below.
+
+However, there is still much to do before Cylc 8 is production-ready. Every
+component needs detailing, extensive testing, and performance optimisation; and
+some important sub-systems are still missing. For example, the primary tree
+workflow status view does not yet perform well enough for large workflows;
+control functionality is not integrated with the view components; we don't have
+efficient incremental data updates flowing through to the UI; some aspects of
+security have not been addressed; and we haven't implemented fine-grained
+authorization yet. For details of what remains to be done, see [Prioritization
+of remaining tasks](#prioritization-of-remaining-tasks).
+
+*We are aiming to have a major preview release of Cylc 8 ready for widespread
+testing in September of this year, and to officially release cylc-8.0 in
+the first quarter of 2021.*
+
+## Progress to date
+
+(First read the [Executive summary](#executive-summary).)
+
+A recent screenshot of the new web UI showing live workflow data:
+![Cylc 8 web UI as of February 2020](img/cylc-8-ui-feb2020.png)
+
+As of February 2020:
+- The previously monolithic Cylc project has been split into multiple modular
+  projects on GitHub (primarily because the UI is no longer Python based)
+- Our new communications platforms Riot (developer chat and video
+  conferencing) and Discourse (mail forum) are proving effective
+- Web UI design settled after extensive consultation with users
```

Comment:
I know we've consulted via Discourse but not all users will have seen this and many won't really understand some of the changes until they get a chance to try it out. I wouldn't want them to think it is fixed in stone. How about something like:
"Web design settled after consultation with users (although we anticipate lots of feedback during user testing)"